plugins/paloalto_wildfire/komand_paloalto_wildfire/actions/submit_file_from_url/action.py

```diff
@@ -33,35 +34,32 @@ def run(self, params={}):
         # 00da: pdf
         # 00fd: --------------------------8557684369749613--
         # ...
+        url = params.get(Input.URL)
+        if Utils.check_link_for_supported_file_type(url):
+            file_from_url = self.connection.client.get_file_from_url(url=url)
+            verdict = self.connection.client.get_verdicts(analysed_hash=hashlib.sha256(file_from_url).hexdigest())
+            if verdict == UNKNOWN_VERDICT:
+                try:
+                    response = xmltodict.parse(self.connection.client.submit_file_from_url(url))
+                    out = dict(response["wildfire"]["upload-file-info"])
+                except PluginException:
+                    self.logger.info("Error occurred")
+                    raise
+                except KeyError:
+                    self.logger.warning(f"The response contain unexpected format. Response {response}")
+                    raise PluginException(
+                        cause="The response did not contain a correct format.",
+                        assistance="Check the logs and if the issue persists please contact support.",
+                    )
 
-        req = {
-            "apikey": (None, self.connection.api_key),
-            "url": (None, params.get("url")),
-        }
-
-        try:
-            r = requests.post(url, files=req)
-            o = xmltodict.parse(r.content)
-            out = dict(o["wildfire"]["upload-file-info"])
-        except:
-            self.logger.info("Error occurred")
-            raise
-
-        if not out["filename"]:
-            out["filename"] = "Unknown"
-
-        return {"submission": out}
+                if not out.get("filename"):
```

Comment:
I will change text for filename and for URL. From my point of view logging is not needed because if this situation appears we will already now in which place it happened by new text for this fields.